src/components/Search.js

```diff
@@ -1,11 +1,12 @@
 import React from 'react'
 
-const Search = () => {
+const Search = ({searchTerm, handleChange}) => {
   return (
     <div className="ui huge fluid icon input">
       <input
         type="text"
-        placeholder={"Search your Recent Transactions"}
+        placeholder={"Search your Recent Transactions"} value={searchTerm}
```

Comment:
don't forget to pass the `value` here so that this is a fully controlled component.